pkg/controllers/offline.go

```diff
@@ -85,6 +85,7 @@ func (s *sccOfflineMode) ReconcileRegisterError(registrationObj *v1.Registration
 	if phase == types.RegistrationMain {
 		v1.RegistrationConditionOfflineRequestReady.SetError(registrationObj, "Failed to update Offline Request secret", registerErr)
 	}
+	registrationObj.SetCurrentCondition(v1.RegistrationConditionOfflineRequestReady)
```

Comment:
Do we need to update the current condition on line 73 and 134?

Reply:
I don't think we strictly need to based on logic I would assume. However, I could see that from a UI perspective that maybe we should set `RegistrationConditionOfflineCertificateReady` as the current condition after line 73 - otherwise it will just be expected to be `Progressing`.

For line 134 - we should be covered by `PrepareSuccessfulActivation` setting the current status to Done after it was successful.